api/GPUQuerySet.json

```diff
@@ -50,9 +54,13 @@
             },
             "edge": "mirror",
             "firefox": {
+              "version_added": "preview",
```

Comment:
This `count` and `type` are in a dictionary `GPUQuerySetDescriptor`, and it is not so obvious these become properties of GPUQuerySet. Question is, if these are properties, why is `pipelineStatistics` not present - should it be?

Reply:
Where are you getting `pipelineStatistics` from? I can't see it in the [editor's draft](https://gpuweb.github.io/gpuweb/) anywhere.